src/libelf.c

```diff
@@ -183,54 +183,37 @@ struct Header
     char trailer[2];
 };
 
-void OmToHeader(Header *h, ObjModule *om)
-{
-    size_t len;
-    if (om->name_offset == -1)
-    {
-        len = strlen(om->name);
-        memcpy(h->object_name, om->name, len);
-        h->object_name[len] = '/';
-    }
-    else
-    {
-        len = sprintf(h->object_name, "/%d", om->name_offset);
-        h->object_name[len] = ' ';
-    }
-    assert(len < OBJECT_NAME_SIZE);
-    memset(h->object_name + len + 1, ' ', OBJECT_NAME_SIZE - (len + 1));
-
-    /* In the following sprintf's, don't worry if the trailing 0
-     * that sprintf writes goes off the end of the field. It will
-     * write into the next field, which we will promptly overwrite
-     * anyway. (So make sure to write the fields in ascending order.)
-     */
-    len = sprintf(h->file_time, "%llu", (longlong)om->file_time);
-    assert(len <= 12);
-    memset(h->file_time + len, ' ', 12 - len);
-
+void OmToHeader(Header *h, ObjModule *om) {
```

Comment:
While there are a few places in DMD, and especially the backend, where this style is used, I believe the current standard is to do it the same way it was originally. (bracket on next line)